### DISCUSSION TITLE
Missing C# features with csharp-roslyn

### DISCUSSION THREAD

**Discussion by GroundCombo** [emacs-lsp/lsp-mode#4616] [Q&A]
I have lsp-mode running with csharp-roslyn and in general it is working surprisingly nicely with Unity projects (I used to work with omnisharp-mode, but it couldn't handle some projects that used source generators). However, there are a couple of strange omissions and I was hoping someone could educate me:

1. Flycheck is reporting no errors. If I for example make an intentional typo, it still shows 0 and nothing is highlighted. I don't see any errors in lsp-log, and flycheck is reporting 'lsp' as the syntax checker (may enable/run). Is this a missing feature or can there be a setting I'm missing? Code actions do include a "fix typo".
2. Is it possible to have whole method signature completion with parameter placeholders and tab/shift-tab support? I understand that this is normally implemented via yasnippet (lsp-enable-snippet is t), but again I'm not knowledgeable enough to understand whether csharp-roslyn is missing something, lsp-roslyn doesn't support the feature or if my configuration is broken.

**Comment by gamedolphin** [comment]
The first one is reported in this issue here - https://github.com/emacs-lsp/lsp-mode/issues/4740
